package.json

```diff
@@ -12,14 +12,25 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+<<<<<<< HEAD
```

Comment:
MERGE Conflicts that need to be resolved

Reply:
resolved